java/configuration/src/test/java/sleeper/configuration/properties/SleeperPropertiesTest.java

```diff
@@ -55,6 +56,20 @@ void shouldGetNumericValueAsString() {
         assertThat(testSleeperProperties.get(PAGE_SIZE)).isEqualTo("5");
     }
 
+    @Test
+    void shouldReturnNullIntegerObjectWhenValueSetToNull() {
+        //Value needs to be assigned to variable as null not directly allowed as method argument in present context
+        String testVal = null;
+        // Given
+        TestSleeperProperties testSleeperProperties = new TestSleeperProperties();
+
+        //When
+        testSleeperProperties.set(GARBAGE_COLLECTOR_LAMBDA_CONCURRENCY_MAXIMUM, testVal);
```

Comment:
This call won't do anything, the set method has a null check. We could just leave this out.

Reply:
Addressed